chef-server-compliance-delivery-devops/azuredeploy.parameters.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
+      "value": "westus"
+    },
+    "vm-adminName": {
+      "value": "chef"
+    },
+    "vm-adminPassword": {
+      "value": "GEN-PASSWORD"
+    },
+    "vm-size": {
+      "value": "Standard_D2"
+    },
+    "dnsLabelPrefix": {
+      "value": "GEN-UNIQUE"
+    },
+    "baseUrl": {
+      "value": "https://raw.githubusercontent.com/uday-sg/azure-quickstart-templates/add-chef-p2p/chef-server-compliance-delivery-devops"
```

Comment:
Is this
https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/chef-server-compliance-delivery-devops